pkg/mandoc

```diff
@@ -13,32 +13,18 @@ desc="manpage viewer. recommended to use instead of sabotage's minimal man prog"
 sed -i 's@pager = "/usr/bin/more -s"@pager = "/bin/less -s"@' main.c
 
 cat << EOF > configure.local
-PREFIX=$butch_prefix
-MANDIR="$butch_prefix"/share/man
-LIBDIR="$butch_prefix"/lib
-SBINDIR="$butch_prefix"/bin
+PREFIX="$butch_prefix"
+MANDIR="${PREFIX}/share/man"
+SBINDIR="${PREFIX}/bin"
+MANPATH_DEFAULT="${PREFIX}/share/man:${PREFIX}/local/share/man"
+MANPATH_BASE="${PREFIX}/share/man"
+UTF8_LOCALE=en_US.UTF-8
+OSNAME="sabotage"
 CFLAGS="$optcflags"
-BUILD_DB=0
-HAVE_DIRENT_NAMLEN=0
-HAVE_FGETLN=1
-HAVE_FTS=0
-HAVE_GETSUBOPT=1
-HAVE_MMAP=1
-HAVE_STRCASESTR=1
-HAVE_STRLCAT=1
-HAVE_STRLCPY=1
-HAVE_STRPTIME=1
-HAVE_STRSEP=1
-HAVE_WCHAR=1
+LDFLAGS="$optldflags"
 EOF
 
-./configure -C --disable-nls $xconfflags
+./configure
 
-make V=1 -j$MAKE_THREADS LDFLAGS="$optldflags"
+make -j$MAKE_THREADS
 make DESTDIR="$butch_install_dir" install
-
-dest="$butch_install_dir""$butch_prefix"
-rm "$dest"/bin/man
-chmod 0755 "$dest"/bin/mandoc
-chmod 0755 "$dest"/bin/demandoc
-ln -sf mandoc "$dest"/bin/man
```

Comment:
mandoc is now available as /bin/man and has the right perms ?

Reply:
yeah. It hard links between the executables and all have +x on them

Reply:
looks like the perms are 555 by default. a little unusual but I see nothing wrong with that

Reply:
well, i guess in that case the chmod's made sense
and the ln -sf made sense too, as i replaced a costly hardlink with a cheap symlink

Reply:
if you do `butch pack` for example, you suddenly have 2 identical copies of the file after `butch unpack`

Reply:
I'm not able to replicate that...

Reply:
you're right, but its not possible to differ between a hardlink and on ordinary file in a directory listing (without controlling it manually with stat), so you could be fooled into thinking that it's a copy whereas a symlink is easily identifiable as such. in short, i prefer symlinks.

Reply:
That makes sense.

Reply:
i dont see these changes !?

Reply:
perhaps because the review is on the removal of those lines and not the addition of the new ones?

Reply:
i'm actually refering to the "replace all the hardlinks" commits